contoso_supermarket/operations/freezer_monitoring/values.yaml

```diff
@@ -0,0 +1,32 @@
+broker:
+  mqttBroker:
+    image:
+      repository: jumpstartagora.azurecr.io/contoso-supermarket/mqtt-broker
+      tag: latest
+    imagePullPolicy: Always
+  ports:
+    - name: http
+      port: 1883
+      targetPort: 1883
+  replicas: 1
+  type: LoadBalancer
+kubernetesClusterDomain: cluster.local
+simulator:
+  mqttSimulator:
+    image:
+      repository: jumpstartagora.azurecr.io/contoso-supermarket/mqtt-simulator
```

Comment:
we should add a placeholder for acr like _acrName_ and add it in the github action "update-files" to replace it

Reply:
These ones are not changing. They will always pull from jumpstartagora.azurecr.io